### PR TITLE
Remove apache commons dependencies

### PR DESCRIPTION
**Pull request by francoismora**

### CODE REVIEW

File: flex-checks/src/main/java/org/sonar/flex/checks/UnusedFunctionParametersCheck.java [hunk 1]
```diff
@@ -140,11 +142,10 @@ private void reportUnusedArgument() {
     }
 
     if (nbUnusedArgs > 0) {
-      addIssue(
-        MessageFormat.format(
-          "Remove the unused function {0} \"{1}\".", nbUnusedArgs > 1 ? "parameters" : "parameter",
-          StringUtils.join(builder.toString().trim().split(" "), ", ")),
-        currentScope.functionDec);
+      String parameter = nbUnusedArgs > 1 ? "parameters" : "parameter";
+      String parameters = Pattern.compile(" ").splitAsStream(builder.toString().trim()).collect(joining(", "));
```
Comment (saberduck) [inline flex-checks/src/main/java/org/sonar/flex/checks/UnusedFunctionParametersCheck.java:146]: I don't understand why we don't build desired format directly in the loop on line 139? It seems like building the string and splitting is entirely unnecessary.

File: flex-checks/src/test/java/org/sonar/flex/checks/ASDocCheckTest.java [hunk 2]
```diff
@@ -46,11 +46,7 @@ protected void activeOnly(String... fieldNames) throws IllegalAccessException {
       if (!f.getType().equals(boolean.class)) {
         continue;
       }
-      if (ArrayUtils.contains(fieldNames, f.getName())) {
-        f.setBoolean(check, true);
-      } else {
-        f.setBoolean(check, false);
-      }
+      f.setBoolean(check, Arrays.asList(fieldNames).contains(f.getName()));
```
Comment (saberduck) [inline flex-checks/src/test/java/org/sonar/flex/checks/ASDocCheckTest.java:49]: :D

File: sonar-flex-plugin/src/main/java/org/sonar/plugins/flex/cobertura/CoberturaReportParser.java [hunk 3]
```diff
@@ -125,8 +124,9 @@ private static void collectFileData(SMInputCursor clazz, NewCoverage newCoverage
 
       String isBranch = line.getAttrValue("branch");
```
Comment (saberduck) [inline sonar-flex-plugin/src/main/java/org/sonar/plugins/flex/cobertura/CoberturaReportParser.java:125]: I think this variable should have `boolean` type directly

File: sonar-flex-plugin/src/test/java/org/sonar/plugins/flex/cobertura/CoberturaSensorTest.java [hunk 4]
```diff
@@ -55,14 +56,7 @@ public void setUp() throws Exception {
 
   @Test
   public void shouldParseReport() throws Exception {
-    String content = new String(Files.readAllBytes(Paths.get(TEST_DIR, "src/example/File.as")), StandardCharsets.UTF_8);
```
Comment (saberduck) [inline sonar-flex-plugin/src/test/java/org/sonar/plugins/flex/cobertura/CoberturaSensorTest.java:58]: I don't see how is this change related to the PR

**Review by saberduck — CHANGES_REQUESTED** [review]

File: flex-checks/src/main/java/org/sonar/flex/checks/UnusedFunctionParametersCheck.java (hunk 1 above)
Comment (francoismora) [reply flex-checks/src/main/java/org/sonar/flex/checks/UnusedFunctionParametersCheck.java]: Changed

File: sonar-flex-plugin/src/test/java/org/sonar/plugins/flex/cobertura/CoberturaSensorTest.java (hunk 4 above)
Comment (francoismora) [reply sonar-flex-plugin/src/test/java/org/sonar/plugins/flex/cobertura/CoberturaSensorTest.java:58]: I extracted the code in a separate method to reuse it in a new unit test.

**Review by saberduck — APPROVED** [review]
